### PR TITLE
Fix plugin for 1.17

### PR DESCRIPTION
**Pull request by GregSutcliffe**
@tbrisker could you review? I'm not sure if this new style is 1.17 only, or ok for 1.16?

### CODE REVIEW

**Comment by ekohl** [comment]
Not entirely required but it's nice if plugins have a table of supported foreman versions with matching plugin versions in their README. Something like https://github.com/theforeman/foreman_bootdisk#compatibility makes life easier for users and packagers.

**Comment by GregSutcliffe** [comment]
@ekohl added. It's been a while since I did any serious testing so this is from memory, but it's a start :)

File: lib/foreman_column_view/engine.rb [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
 
-      # Extend core HostHelper to add rows to Properties on hosts/show
-      ::HostsHelper.send :include, ForemanColumnView::HostsHelperExtension
-    end
+			# Extend core HostHelper to add rows to Properties on hosts/show
+			::HostsHelper.send :include, ForemanColumnView::HostsHelperExtension
+		end
 
-  end
+	end
```
Comment (bastilian) [inline lib/foreman_column_view/engine.rb]: Tabs vs. Spaces. ;)

**Comment by GregSutcliffe** [comment]
@bastilian updated :)

**Comment by GregSutcliffe** [comment]
Merged, thanks both. will cut a release.

**Pull request merged**